accounts/models.py

```diff
@@ -0,0 +1,92 @@
+from allauth.account.signals import user_signed_up
+from django.dispatch import receiver
+from django.contrib.auth.models import User
+from django.db import models
+from django.conf import settings
+from django.db.models.signals import post_delete
+from .lists import user_types, lms_modules
+
+
+class Profile(models.Model):
+    """
+    Define Profile Model with OneToOne relationship to AUTH_USER_MODEL and
+    custom fields to suit Signup Form Extending in forms.py replicating same
+    in DB.
+    Using "related_name" in the OneToOne relationship between the two models
+    specifies the reverse relationship to the User Model, allowing us to
+    target the custom fields for injection into the profile.html
+    template via `{{ user.profile.<<field_name>> }}`.
+    """
+    user = models.OneToOneField(
+        settings.AUTH_USER_MODEL, default=1,
+        related_name='profile',
+        on_delete=models.CASCADE
+    )
+    slack_display_name = models.CharField(
+        max_length=80,
+        blank=False,
```

Comment:
Is there a reason you are allowing for null, but not for blank values?

Reply:
As per Django reference it is not recommended to set `null=True`, but rather use an empty string. See more info here: https://docs.djangoproject.com/en/3.1/ref/models/fields/#null
(btw this was news to me too until somebody highlighted it to me in a PR on this project)

Reply:
Same goes for the below fields.

Reply:
I don't think the `null=''` will work. I'd add `default=''` instead.

Reply:
Just fixed that and pushed with through with migrations made too @stefdworschak